zio-http/src/main/scala/zhttp/service/Handler.scala

```diff
@@ -142,7 +142,7 @@ private[zhttp] final case class Handler[R](
                 }
               case None        =>
                 UIO {
-                  writeResponse(Response.status(Status.NOT_FOUND), jReq)
+                  writeResponse(Response.status(Status.Not_Found), jReq)
```

Comment:
```suggestion
                  writeResponse(Response.status(Status.NotFound), jReq)
```

Underscores should be dropped.

Reply:
I have removed the underscores from all Status names.